mlptrain/sampling/plumed.py

```diff
@@ -1033,6 +1033,43 @@ def __init__(self, name: str, atom_groups: Sequence = None):
         )
 
 
+class PlumedCNCV(_PlumedCV):
+    def __init__(self, name: str, ref: float, atom_groups: Sequence = None):
+        """
+        PLUMED collective variable as a coordinattion number,
```

Comment:
We need to prive more description, since the CN is less intuitive than distance or angle. For instance
```suggestion
        PLUMED collective variable as a coordination number (CN) between two atoms or groups of atoms
```